crates/cxx-qt-build/src/lib.rs

```diff
@@ -46,6 +46,13 @@ use cxx_qt_gen::{
     GeneratedRustBlocks, Parser,
 };
 
+// On Unix platforms, included files are symlinked into destination folders.
+// On non-Unix platforms, due to poor support for symlinking, included files are deep copied.
+#[cfg(unix)]
+const INCLUDE_VERB: &str = "create symlink";
+#[cfg(not(unix))]
+const INCLUDE_VERB: &str = "deep copy files";
```

Comment:
nice way of expressing this :+1: 

However, a small nitpick: Please move this constant into the `dir` module, as it's tightly coupled with the behavior of the `symlink_or_copy_directory` function.